File: PF_Custome_CollectionView/PF_Custome_CollectionView/PFCollectionViewCell.m
//
//  PFCollectionViewCell.m
//  PF_Custome_CollectionView
//
//  Created by 天下林子 on 2019/6/21.
//  Copyright © 2019 linhaoCui. All rights reserved.
//

#import "PFCollectionViewCell.h"


@interface PFCollectionViewCell ()

@end


@implementation PFCollectionViewCell

-(UILabel *)titleLb
{
    if (!_titleLb) {
        _titleLb = [UILabel new];
        [self addSubview:_titleLb];
        _titleLb.frame = CGRectMake(0, 0, self.bounds.size.width, 50);
        _titleLb.text = @"rrrrrr";
        
        
        _titleLb.font = [UIFont systemFontOfSize:15];
        _titleLb.textAlignment = NSTextAlignmentCenter;
        
    }
    return _titleLb;
    
}

@end

File: PF_Custome_CollectionViewCell_swift/PF_Custome_CollectionViewCell/PFCollectionViewCell.swift
//
//  PFCollectionViewCell.swift
//  PF_Custome_CollectionViewCell
//
//  Created by 天下林子 on 2019/6/24.
//  Copyright © 2019 linhaoCui. All rights reserved.
//

import UIKit


class PFCollectionViewCell: UICollectionViewCell {
	
	var label  = UILabel()
	
	override init(frame: CGRect) {
		super.init(frame: frame)
		
		initUI()
	}
	
	required init?(coder aDecoder: NSCoder) {
		fatalError("init(coder:) has not been implemented")
	}
	
	func initUI() {
		
		self.addSubview(label)
		label.frame = CGRect(x: 0, y: 0, width: self.bounds.size.width, height: 50)
		label.text = "飞龙在天"
		label.font = UIFont.systemFont(ofSize: 15)
		label.textAlignment = .center
	}
	
}
